docs/docs/components/reanimated_swipeable.md

```diff
@@ -0,0 +1,215 @@
+```
+
+## Properties
+
+### `friction`
+
+a number that specifies how much the visual interaction will be delayed compared to the gesture distance. e.g. value of 1 will indicate that the swipeable panel should exactly follow the gesture, 2 means it is going to be two times "slower".
+
+### `leftThreshold`
+
+distance from the left edge at which released panel will animate to the open state (or the open panel will animate into the closed state). By default it's a half of the panel's width.
+
+### `rightThreshold`
+
+distance from the right edge at which released panel will animate to the open state (or the open panel will animate into the closed state). By default it's a half of the panel's width.
+
+### `dragOffsetFromLeftEdge`
+
+distance that the panel must be dragged from the left edge to be considered a swipe. The default value is 10.
+
+### `dragOffsetFromRightEdge`
+
+distance that the panel must be dragged from the right edge to be considered a swipe. The default value is 10.
+
+### `overshootLeft`
+
+a boolean value indicating if the swipeable panel can be pulled further than the left actions panel's width. It is set to `true` by default as long as the left panel render method is present.
+
+### `overshootRight`
+
+a boolean value indicating if the swipeable panel can be pulled further than the right actions panel's width. It is set to `true` by default as long as the right panel render method is present.
+
+### `overshootFriction`
+
+a number that specifies how much the visual interaction will be delayed compared to the gesture distance at overshoot. Default value is 1, it mean no friction, for a native feel, try 8 or above.
+
+### `onSwipeableOpen`
+
+method that is called when action panel gets open (either right or left). Takes swipe direction as
+an argument.
+
+### `onSwipeableClose`
+
+method that is called when action panel is closed. Takes swipe direction as
+an argument.
+
+### `onSwipeableWillOpen`
+
+method that is called when action panel starts animating on open (either right or left). Takes swipe direction as
+an argument.
+
+### `onSwipeableWillClose`
+
+method that is called when action panel starts animating on close. Takes swipe direction as
+an argument.
+
+### `renderLeftActions`
+
+method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
+This map describes the values to use as inputRange for extra interpolation:
+AnimatedValue: [startValue, endValue]
+
+progressAnimatedValue: [0, 1]
+dragAnimatedValue: [0, +]
+
+To support `rtl` flexbox layouts use `flexDirection` styling.
+
+### `renderRightActions`
+
+method that is expected to return an action panel that is going to be revealed from the right side when user swipes left.
+This map describes the values to use as inputRange for extra interpolation:
+AnimatedValue: [startValue, endValue]
+
+progressAnimatedValue: [0, 1]
+dragAnimatedValue: [0, -]
```

Comment:
Those are shared values not animated values in the new implementation

Reply:
replaced those instances